<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorRenderingOptionsWidgetBase</class>
 <widget class="QWidget" name="QgsVectorRenderingOptionsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Vector Rendering Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mSimplifyDrawingGroupBox">
     <property name="title">
      <string>Enable feature si&amp;mplification by default for newly added layers</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="_14" columnstretch="0,1,0">
      <item row="1" column="1">
       <widget class="QgsDoubleSpinBox" name="mSimplifyDrawingSpinBox">
        <property name="toolTip">
         <string>Higher values result in more simplification</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mSimplifyAlgorithmLabel">
        <property name="toolTip">
         <string>This algorithm is only applied to simplify on local side</string>
        </property>
        <property name="text">
         <string>Simplification algorithm</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QgsScaleComboBox" name="mSimplifyMaximumScaleComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mSimplifyMaxScaleLabel">
        <property name="text">
         <string>Maximum scale at which the layer should be simplified (1:1 always simplifies)</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="mSimplifyDrawingAtProvider">
        <property name="text">
         <string>Simplify on provider side if possible</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_59">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; Feature simplification may speed up rendering but can result in rendering inconsistencies</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="mSimplifyAlgorithmComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_65">
        <property name="text">
         <string>Simplification threshold (higher values result in more simplification)</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mSimplifyDrawingPx">
        <property name="text">
         <string>pixels</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mSegmentationGroupBox">
     <property name="title">
      <string>Curve Segmentation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_20">
      <item row="0" column="0">
       <widget class="QLabel" name="mSegmentationToleranceLabel">
        <property name="text">
         <string>Segmentation tolerance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="mSegmentationToleranceSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mToleranceTypeLabel">
        <property name="text">
         <string>Tolerance type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mToleranceTypeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>225</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsScaleComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsscalecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mSimplifyDrawingGroupBox</tabstop>
  <tabstop>mSimplifyDrawingSpinBox</tabstop>
  <tabstop>mSimplifyAlgorithmComboBox</tabstop>
  <tabstop>mSimplifyDrawingAtProvider</tabstop>
  <tabstop>mSimplifyMaximumScaleComboBox</tabstop>
  <tabstop>mSegmentationToleranceSpinBox</tabstop>
  <tabstop>mToleranceTypeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
